### ISSUE TITLE
CSS changes reduce readability of documents

### ISSUE THREAD

**Issue by pfps** [w3c/rdf-star-wg#61]
Many working group documents have had CSS changes applied to them, for example https://github.com/w3c/rdf-turtle/pull/20, which include code like:

```
    @media (max-width: 850px) {
        table th, table td { font-size: 12px; padding: 3px 4px;}
        table.ex th, table.ex td {overflow-x: scroll; max-width: 42vw !important;}
```

This change has the effect of reducing the side of text in tables when window width falls below a 850px, which occurs in many situations, not just on small devices.  The reduced size makes it harder to read tables.  In at least some cases there is no benefit to reducing the text size as the table can be rendered with no problems without reducing the text size.

Here is  a screenshot showing the size difference.  

![Screenshot from 2023-05-20 08-12-28](https://github.com/w3c/rdf-star-wg/assets/6618235/e8720234-b493-4485-aef3-940a5b7fe503)

Note how when the body text is small but easily readable the text in the table is less readable.   There is no reason to reduce the table text size here as the table can be rendered at this width with no problems.

**Comment by TallTed** [comment]
I agree with @pfps that this dynamic font size change is problematic. Barring revelation of a strong justification for it, I think this aspect of the "improving display on mobile phones" CSS changes, which I do not think it accomplishes, should be undone/removed.

**Comment by pchampin** [comment]
I believe that the problematic changes have been reverted.
Is it ok to close this issue?

**Comment by pfps** [comment]
Have they?  What PRs?

**Comment by pfps** [comment]
That table looks good.   What about other tables and other documents?  The change went in to many documents, with maybe only Semantics being excepted.

**Comment by TallTed** [comment]
I use Web Developer extension in Chrome, which has Web Developer Responsive Layouts, which show how the same web page will look in variously sized viewports, smaller and larger than the browser window they're loading in. This was very helpful in checking this issue circa May 2023.

When I try to check https://www.w3.org/TR/rdf12-turtle/#abbrev now, none of the viewports populate, and I get a message that says, "If the responsive layouts are not loading it is because the website is blocking the ability for it to be loaded in an iframe."

Does anyone know of some magic that can make loading in an iframe work?

**Comment by pchampin** [comment]
Ok, the changes have been reverted in RDF-Semantics, and RDF-Concepts, from what I can see, has no CSS rule altering table based on `@media` selectors. But in other specs, such rules are still present.

Should we leave it to the editor of each document what they want to do with this?
